docs/ref/signals.txt

```diff
@@ -344,6 +344,12 @@ Argument        Value
 ``using``       ``"default"`` (since the default router sends writes here)
 ==============  ============================================================
 
+__NOTE__:
```

Comment:
Also the markup here needs to be: 

```
.. note::

    Multiple calls...
```

Reply:
Oops, I'm sorry, I was traveling and jetlagged and used markdown just out of habit.

I've just made the changes.

Reply:
No issues. :-)